crates/snforge-scarb-plugin/src/attributes/test.rs

```diff
@@ -37,7 +37,21 @@ fn test_internal(
 ) -> Result<TokenStream, Diagnostics> {
     assert_is_used_once::<TestCollector>(db, func)?;
     args.assert_is_empty::<TestCollector>()?;
-    ensure_parameters_only_with_fuzzer_attribute(db, func)?;
+    ensure_parameters_only_with_fuzzer_or_test_case_attribute(db, func)?;
+
+    let has_test_case = has_test_case_attribute(db, func);
+    let has_fuzzer = has_fuzzer_attribute(db, func);
+
+    // If the function has `#[test]` attribute and does not have `#[fuzzer]`, we can
```

Comment:
Shouldn't it be test_case?
```suggestion
    // If the function has `#[test_case]` attribute and does not have `#[fuzzer]`, we can
```

Reply:
Good catch 👁️